qrcode/image/pil.py

```diff
@@ -11,20 +14,17 @@ class PilImage(qrcode.image.base.BaseImage):
 
     def new_image(self, **kwargs):
         if not Image:
-            raise ImportError("PIL library not found.")
+            msg = "PIL library not found."
+            raise ImportError(msg)
 
         back_color = kwargs.get("back_color", "white")
         fill_color = kwargs.get("fill_color", "black")
 
-        try:
+        with contextlib.suppress(AttributeError):
             fill_color = fill_color.lower()
-        except AttributeError:
-            pass
 
-        try:
+        with contextlib.suppress(AttributeError):
             back_color = back_color.lower()
-        except AttributeError:
-            pass
```

Comment:
Ditto about `suppress` here...

Reply:
It's because fill/back_color can also be a tuple.

``` ["TransParent", "red", (255, 195, 235)]```